doc-content/enterprise-only/getting-started/business-process-proc.adoc

```diff
@@ -13,8 +13,7 @@ You must delete the sample business process before you can create your own.
 .Procedure
 . Log in to {CENTRAL} and click *Menu* -> *Design* -> *Projects*, then click *Mortgage Process*.
 . Input `MortgageApprovalProcess.bpmn` in to the project's asset search box and click on the *MortgageApprovalProcess* business process.
-. Click *Delete* in the upper-right corner, then click *Proceed* in the *Usages notification* window.
-. Click *Delete* to confirm that you want to delete this asset.
+. Click *Delete* in the upper-right corner, then click *Delete* to confirm that you want to delete this asset.
```

Comment:
There's a lot of options in that toolbar, which places `Delete` button in particular almost in the middle rather than to the right. Maybe changing the instruction to something more precise?
`Select Delete from the toolbar ` or something along these lines.